src/store/modules/blocks/state.js

```diff
@@ -3,6 +3,8 @@
  */
 export default {
   height: 0,
+  generation: {},
   block: {},
-  blocks: []
+  blocks: [],
+  generations: []
```

Comment:
If we mostly renamings `blocks` into `generations`, maybe we can drop `blocks` property?